src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -125,9 +126,11 @@ def _parse_addr(value):
         #   Group 1: [
         #   Group 2: 2001:db8:85a3::8a2e:370:7334
         addr = match.group(3) if match.group(3) else match.group(2)
-        if not is_valid_ip_address(addr):
-            raise LookupError('Invalid RGW address \'{}\' found'.format(addr))
-        return addr
+        try:
+            if ipaddress.ip_address(addr):
+                return True
```

Comment:
```suggestion
                return addr
```

Reply:
@votdev thanks for pointing to this. I did the changes accordingly.